src/renderer/components/channel-blocker-settings/channel-blocker-settings.js

```diff
@@ -0,0 +1,58 @@
+import Vue from 'vue'
+import { mapActions } from 'vuex'
+import FtToggleSwitch from '../ft-toggle-switch/ft-toggle-switch.vue'
+import FtInput from '../ft-input/ft-input.vue'
+import FtFlexBox from '../ft-flex-box/ft-flex-box.vue'
+import ChannelBlockerSettingsList from '../channel-blocker-settings-list/channel-blocker-settings-list.vue'
+
+export default Vue.extend({
+  name: 'ChannlBlockerSettings',
+  components: {
+    'ft-toggle-switch': FtToggleSwitch,
+    'ft-input': FtInput,
+    'ft-flex-box': FtFlexBox,
+    'channel-blocker-settings-list': ChannelBlockerSettingsList
+  },
+  data: function () {
+    return {
+      hasQuery: false
+    }
+  },
+  computed: {
+    useChannelBlocker: function () {
+      return this.$store.getters.getUseChannelBlocker
+    },
+
+    channelBlockerCache: function () {
+      if (!this.hasQuery) {
+        return this.$store.getters.getChannelBlockerCache
+      } else {
+        return this.$store.getters.getSearchChannelBlockerCache
+      }
+    }
+  },
+  methods: {
+    handleUpdateChannelBlocker: function (value) {
+      this.updateUseChannelBlocker(value)
```

Comment:
Is this feature really something that needs to be toggled on / off? I thought something was broken at first because I didn't see any Block option like in your screenshots. This feels unneeded and should always be considered as "Enabled"